packages/twenty-server/src/engine/core-modules/actor/utils/build-created-by-from-auth-context.util.ts

```diff
@@ -0,0 +1,65 @@
+import { Logger } from '@nestjs/common';
+
+import { isDefined } from 'twenty-shared';
+
+import { buildCreatedByFromApiKey } from 'src/engine/core-modules/actor/utils/build-created-by-from-api-key.util';
+import { buildCreatedByFromFullNameMetadata } from 'src/engine/core-modules/actor/utils/build-created-by-from-full-name-metadata.util';
+import { AuthContext } from 'src/engine/core-modules/auth/types/auth-context.type';
+import { ActorMetadata } from 'src/engine/metadata-modules/field-metadata/composite-types/actor.composite-type';
+import { TwentyORMGlobalManager } from 'src/engine/twenty-orm/twenty-orm-global.manager';
+import { WorkspaceMemberWorkspaceEntity } from 'src/modules/workspace-member/standard-objects/workspace-member.workspace-entity';
+
+type BuildCreatedByFromAuthContextArgs = {
+  authContext: AuthContext;
+  twentyORMGlobalManager: TwentyORMGlobalManager;
+  logger: Logger;
+};
+export const buildCreatedByFromAuthContext = async ({
+  authContext,
+  twentyORMGlobalManager,
+  logger,
+}: BuildCreatedByFromAuthContextArgs): Promise<ActorMetadata> => {
+  const { workspace, workspaceMemberId, user, apiKey } = authContext;
+
+  switch (true) {
```

Comment:
new pattern, why not but we usually use early if return

Reply:
Indeed no real + value here